gatsby-node.js

```diff
@@ -2,75 +2,82 @@ const path = require("path")
 const _ = require("lodash")
 const { createFilePath } = require("gatsby-source-filesystem")
 
-exports.createPages = ({ graphql, actions }) => {
+exports.createPages = async ({ graphql, actions, reporter }) => {
   const { createPage } = actions
-
-  return new Promise((resolve, reject) => {
-    const blogPost = path.resolve("./src/templates/blog-post.tsx")
-    const catePage = path.resolve("./src/templates/category-template.tsx")
-
-    resolve(
-      graphql(
-        `
-          {
-            allMarkdownRemark(
-              sort: { fields: [frontmatter___date], order: DESC }
-              limit: 1000
-            ) {
-              group(field: frontmatter___categories) {
-                fieldValue
+  const result = await graphql(
+    `
+      {
+        allMarkdownRemark(
+          sort: { fields: [frontmatter___date], order: DESC }
+          limit: 1000
+        ) {
+          categories: group(field: frontmatter___categories) {
+            fieldValue
+          }
+          authors: group(field: frontmatter___author) {
+            fieldValue
```

Comment:
#minor but at this point it feels like it might make sense to have more field aliases like `name` instead of `fieldName` - so you would be saying `category.name` or `author.name` - what do you think? It's mostly a matter of personal taste.